src/kernel/capinv.c

```diff
@@ -277,7 +278,8 @@ kmem_deact_post(unsigned long *pte, unsigned long old_v)
 	u32_t new_v;
 
 	/* Unset coskmem bit. Release the kmem frame. */
-	new_v = old_v & (~PGTBL_COSKMEM);
+        new_v = chal_pgtbl_flag_clr(old_v, PGTBL_COSKMEM);
```

Comment:
Looks like there is a unnecessary space before `new_v`.

Reply:
fixed